### PR TITLE
🚚 update: Add 110th CatchUp Summary

### PR DESCRIPTION
**Pull request by dheerajdlalwani**

### CODE REVIEW

File: summary/sessions/110/attendees.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+. link:https://twitter.com/TmPreet[Preet Parekh^]
+. link:https://twitter.com/rishit_dagli[Rishit Dagli^]
+. link:https://twitter.com/0xSaurabh[Saurabh Kumar Suryan^]
+. link:https://twitter.com/Darth_Sid512[Siddharth Bhatia^]
+. link:https://twitter.com/skxrxn[Sreekaran Srinath^]
+. link:https://twitter.com/tusharnankanii[Tushar Nankani^]
+. link:https://twitter.com/code_magician[Viranchee Lotia^]
+. link:https://twitter.com/WilfredAlmeida_[Wilfred Almeida]
+. link:https://twitter.com/furtado_jaden[Jaden Furtado^]
+
```
Comment (PranavDani) [inline summary/sessions/110/attendees.adoc]: Delete this line.

File: summary/sessions/110/attendees.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+. Fardeeen Samir D
+. Harshit Raheja
+. Jainam Jagani
+. Jivesh Bhojwani
+. Jia Harisinghani
+. Sagnik Tarafdar
```
Comment (PranavDani) [inline summary/sessions/110/attendees.adoc:27]: Add one line after this.

Comment (PranavDani) [inline summary/sessions/110/attendees.adoc:27]: Add one line after this.

File: summary/sessions/110/content.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Date: 17-12-2022
+
+Duration: 4 hrs 24 mins
+
+==== Topics Discussed
+
+* General Introductions.
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^] spoke about link:https://en.wikipedia.org/wiki/Stable_Diffusion[Stable Diffusion^].
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^] also spoke about link:https://wasi.dev[WASI - The WebAssembly System Interface^].
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^]
```
Comment (PranavDani) [inline summary/sessions/110/content.adoc]: Why is this here?

File: summary/sessions/110/content.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+* link:https://twitter.com/DhiruCodes[Dheeraj Lalwani^] asked what does it mean to run an application or a server "as a service" and how to restart a link:https://nodejs.org[Node.js^] server which is running as a process.
+    ** link:https://twitter.com/furtado_jaden[Jaden Furtado^] replied said that it is used to make the servers run as a background process and to avoid deadlocks.
```
Comment (PranavDani) [inline summary/sessions/110/content.adoc]: ~~replied~~

File: summary/sessions/110/content.adoc [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    ** link:https://pm2.keymetrics.io[PM2 - A process manager for Node.js]
```
Comment (PranavDani) [inline summary/sessions/110/content.adoc:12]: Elaborate.

File: summary/sessions/110/content.adoc [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    ** Command to restart a service: pm2 restart <application name>
```
Comment (PranavDani) [inline summary/sessions/110/content.adoc:13]:
```
pm2 restart <application name>
```

File: summary/sessions/110/content.adoc [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+* link:https://www.linkedin.com/in/mhshetty[Mohit Shetty^] spoke about his company's plan for secure phones.
```
Comment (PranavDani) [inline summary/sessions/110/content.adoc:14]: Mention the company's name if possible.

File: summary/sessions/110/content.adoc [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    ** link:https://twitter.com/SirusTweets[Darshan Rander^] suggested that they also take into consideration hardware level security.
+    ** link:https://twitter.com/SirusTweets[Darshan Rander^] suggested that the issue is getting requests on an Android Device
```
Comment (PranavDani) [inline summary/sessions/110/content.adoc:16]: What issue was being discussed?

File: summary/sessions/110/content.adoc [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+    ** link:https://twitter.com/furtado_jaden[Jaden Furtado^] shared a news article link:https://news.umich.edu/smartphone-security-hole-open-port-backdoors-are-widespread[Smartphone security hole: 'Open port' backdoors are widespread^].
+* link:https://twitter.com/SirusTweets[Darshan Rander^] shared an interesting link:https://twitter.com/drcoderz/status/1601592574027841537[tweet^] by link:https://twitter.com/drcoderz[Roman Jaquez^] which showcases link:https://flutter.dev[Flutter^] apps being run on various "unconventional" displays.
+*  link:https://twitter.com/t3_pat[Pratik Thakare^] spoke about link:https://www.cloudflare.com/learning/network-layer/what-is-tunneling[Tunneling^].
+    ** link:https://www.cloudflare.com/products/tunnel[Cloudflare Tunnel^].
```
Comment (PranavDani) [inline summary/sessions/110/content.adoc:20]: What's the role of this point wrt to the previous one?

File: summary/sessions/110/content.adoc [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+* We then helped link:https://twitter.com/code_magician[Viranchee Lotia^] with some link:https://www.docker.com[Docker^] config.
+* link:https://twitter.com/t3_pat[Pratik Thakare^] then spoke about link:https://www.cloudflare.com/learning/bots/what-is-rate-limiting[Rate Limiter^] on link:https://en.wikipedia.org/wiki/Load_balancing_(computing)[Load Balancer].
```
Comment (PranavDani) [inline summary/sessions/110/content.adoc]: [Load Balancer^]

File: summary/sessions/110/content.adoc [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+* We talked about why we should learn networking.
```
Comment (PranavDani) [inline summary/sessions/110/content.adoc:23]: And why is that?

Comment (PranavDani) [reply summary/sessions/110/content.adoc:23]: Add a line at the end.

File: summary/sessions/110/attendees.adoc (hunk 1 above)
Comment (tusharnankani) [inline summary/sessions/110/attendees.adoc:22]:
```suggestion
. link:https://twitter.com/furtado_jaden[Jaden Furtado^]
```

File: summary/sessions/110/content.adoc (hunk 3 above)
Comment (tusharnankani) [inline summary/sessions/110/content.adoc:10]:
```suggestion
* link:https://twitter.com/rishit_dagli[Rishit Dagli^] spoke about link:https://en.wikipedia.org/wiki/Stable_Diffusion[Stable Diffusion^].
* link:https://twitter.com/rishit_dagli[Rishit Dagli^] also spoke about link:https://wasi.dev[WASI - The WebAssembly System Interface^].
```

File: summary/sessions/110/content.adoc (hunk 4 above)
Comment (tusharnankani) [inline summary/sessions/110/content.adoc:12]:
```suggestion
    ** link:https://twitter.com/furtado_jaden[Jaden Furtado^] replied that it is used to make the servers run as a background process and to avoid deadlocks.
```

File: summary/sessions/110/content.adoc (hunk 11 above)
Comment (tusharnankani) [inline summary/sessions/110/content.adoc:24]:
```suggestion
*  link:https://twitter.com/t3_pat[Pratik Thakare^] spoke about link:https://www.cloudflare.com/learning/network-layer/what-is-tunneling[Tunneling^].
    ** link:https://www.cloudflare.com/products/tunnel[Cloudflare Tunnel^].
* We helped link:https://twitter.com/code_magician[Viranchee Lotia^] with some link:https://www.docker.com[Docker^] config.
* link:https://twitter.com/t3_pat[Pratik Thakare^] spoke about link:https://www.cloudflare.com/learning/bots/what-is-rate-limiting[Rate Limiter^] on link:https://en.wikipedia.org/wiki/Load_balancing_(computing)[Load Balancer].
* We talked about why we should learn networking.
```

File: summary/sessions/110/attendees.adoc [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+. link:https://twitter.com/0xSaurabh[Saurabh Kumar Suryan^]
+. link:https://twitter.com/Darth_Sid512[Siddharth Bhatia^]
+. link:https://twitter.com/skxrxn[Sreekaran Srinath^]
+. link:https://twitter.com/tusharnankanii[Tushar Nankani^]
+. link:https://twitter.com/code_magician[Viranchee Lotia^]
+. link:https://twitter.com/WilfredAlmeida_[Wilfred Almeida]
+. link:https://twitter.com/furtado_jaden[Jaden Furtado^]
+
+. Fardeeen Samir D
+. Harshit Raheja
```
Comment (tusharnankani) [inline summary/sessions/110/attendees.adoc:24]:
```suggestion
. link:https://twitter.com/furtado_jaden[Jaden Furtado^]
. Fardeeen Samir D
. Harshit Raheja
```

File: summary/sessions/110/content.adoc (hunk 7 above)
Comment (tusharnankani) [reply summary/sessions/110/content.adoc:14]: Did not mention.

File: summary/sessions/110/content.adoc [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+
+==== Topics Discussed
+
+* General Introductions.
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^] spoke about link:https://en.wikipedia.org/wiki/Stable_Diffusion[Stable Diffusion^].
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^] also spoke about link:https://wasi.dev[WASI - The WebAssembly System Interface^].
+* link:https://twitter.com/DhiruCodes[Dheeraj Lalwani^] asked what does it mean to run an application or a server "as a service" and how to restart a link:https://nodejs.org[Node.js^] server which is running as a process.
+    ** link:https://twitter.com/furtado_jaden[Jaden Furtado^] replied that it is used to make the servers run as a background process and to avoid deadlocks.
+    ** link:https://pm2.keymetrics.io[PM2 - A process manager for Node.js]
+    ** Command to restart a service: pm2 restart <application name>
```
Comment (tusharnankani) [inline summary/sessions/110/content.adoc:13]:
```suggestion
    ** Command to restart a service: `pm2 restart <application name>`
```

File: summary/sessions/110/attendees.adoc [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+. link:https://twitter.com/tusharnankanii[Tushar Nankani^]
+. link:https://twitter.com/code_magician[Viranchee Lotia^]
+. link:https://twitter.com/WilfredAlmeida_[Wilfred Almeida]
+. link:https://twitter.com/furtado_jaden[Jaden Furtado^]
+. Fardeeen Samir D
+. Harshit Raheja
+. Jainam Jagani
+. Jivesh Bhojwani
+. Jia Harisinghani
+. Sagnik Tarafdar
```
Comment (tusharnankani) [inline summary/sessions/110/attendees.adoc:27]:
```suggestion
. Sagnik Tarafdar

```

File: summary/sessions/110/content.adoc [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+Duration: 4 hrs 24 mins
+
+==== Topics Discussed
+
+* General Introductions.
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^] spoke about link:https://en.wikipedia.org/wiki/Stable_Diffusion[Stable Diffusion^].
+* link:https://twitter.com/rishit_dagli[Rishit Dagli^] also spoke about link:https://wasi.dev[WASI - The WebAssembly System Interface^].
+* link:https://twitter.com/DhiruCodes[Dheeraj Lalwani^] asked what does it mean to run an application or a server "as a service" and how to restart a link:https://nodejs.org[Node.js^] server which is running as a process.
+    ** link:https://twitter.com/furtado_jaden[Jaden Furtado^] replied that it is used to make the servers run as a background process and to avoid deadlocks.
+    ** link:https://pm2.keymetrics.io[PM2 - A process manager for Node.js]
```
Comment (tusharnankani) [inline summary/sessions/110/content.adoc:12]:
```suggestion
    ** link:https://pm2.keymetrics.io[PM2 - A process manager for Node.js] - PM2 is a daemon process manager that will help you manage and keep your application online.
```

**Review by tusharnankani — COMMENTED** [review]
fixed.

File: summary/sessions/110/attendees.adoc (hunk 2 above)
Comment (tusharnankani) [reply summary/sessions/110/attendees.adoc]: Done.

Comment (tusharnankani) [reply summary/sessions/110/attendees.adoc]: Done.

File: summary/sessions/110/content.adoc (hunk 5 above)
Comment (tusharnankani) [reply summary/sessions/110/content.adoc]: DOne.

File: summary/sessions/110/content.adoc (hunk 6 above)
Comment (tusharnankani) [reply summary/sessions/110/content.adoc]: Done.

File: summary/sessions/110/content.adoc (hunk 9 above)
Comment (tusharnankani) [reply summary/sessions/110/content.adoc:20]: Another link^

File: summary/sessions/110/content.adoc [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+    ** link:https://twitter.com/furtado_jaden[Jaden Furtado^] shared a news article link:https://news.umich.edu/smartphone-security-hole-open-port-backdoors-are-widespread[Smartphone security hole: 'Open port' backdoors are widespread^].
+* link:https://twitter.com/SirusTweets[Darshan Rander^] shared an interesting link:https://twitter.com/drcoderz/status/1601592574027841537[tweet^] by link:https://twitter.com/drcoderz[Roman Jaquez^] which showcases link:https://flutter.dev[Flutter^] apps being run on various "unconventional" displays.
+*  link:https://twitter.com/t3_pat[Pratik Thakare^] spoke about link:https://www.cloudflare.com/learning/network-layer/what-is-tunneling[Tunneling^].
+    ** link:https://www.cloudflare.com/products/tunnel[Cloudflare Tunnel^].
+* We helped link:https://twitter.com/code_magician[Viranchee Lotia^] with some link:https://www.docker.com[Docker^] config.
+* link:https://twitter.com/t3_pat[Pratik Thakare^] spoke about link:https://www.cloudflare.com/learning/bots/what-is-rate-limiting[Rate Limiter^] on link:https://en.wikipedia.org/wiki/Load_balancing_(computing)[Load Balancer].
```
Comment (tusharnankani) [inline summary/sessions/110/content.adoc:22]:
```suggestion
* link:https://twitter.com/t3_pat[Pratik Thakare^] spoke about link:https://www.cloudflare.com/learning/bots/what-is-rate-limiting[Rate Limiter^] on link:https://en.wikipedia.org/wiki/Load_balancing_(computing)[Load Balancer^].
```

File: summary/sessions/110/content.adoc (hunk 10 above)
Comment (tusharnankani) [reply summary/sessions/110/content.adoc]: Done.

File: summary/sessions/110/content.adoc (hunk 11 above)
Comment (tusharnankani) [reply summary/sessions/110/content.adoc:23]: @SirusCodes

File: summary/sessions/110/content.adoc (hunk 8 above)
Comment (tusharnankani) [reply summary/sessions/110/content.adoc:16]: @SirusCodes
